lms/djangoapps/course_home_api/outline/serializers.py

```diff
@@ -54,6 +54,7 @@ def get_blocks(self, block):  # pylint: disable=missing-function-docstring
                 'resume_block': block.get('resume_block', False),
                 'type': block_type,
                 'has_scheduled_content': block.get('has_scheduled_content'),
+                'optional_content': block.get('optional_content'),
```

Comment:
Please read this:
https://github.com/openedx/edx-platform/blob/b1f6a464c073409ab9bb31396648d7ed794ec43a/openedx/core/djangoapps/content/block_structure/transformer.py#L16-L57

Reply:
@Agrendalath  so based on that I should bump the block_completion transformer write version, and after we're sure all blocks have the new version, we bump the read version?